core/shared/src/main/scala/zio/NonEmptyOps.scala

```diff
@@ -15,7 +17,11 @@ trait NonEmptyOps[+A, CC[+_], EC[+_]] {
   def last: A
   def map[B](f: A => B): CC[B]
   def reduce[B >: A](op: (B, B) => B): B
+  def size: Int = toIterable.size
   def tail: EC[A]
+  def toArray[B >: A: ClassTag]: Array[B] = toIterable.toArray
```

Comment:
Would it not make more sense to have this as?
```suggestion
  def toArray[B >: A: ClassTag]: Array[B] = iterator.toArray
```

Reply:
Both are inefficient for the default imp.
They should just always be overridden.

Reply:
Sure, but isn't it still possible for them to be called as written?